README.md

```diff
@@ -59,6 +59,22 @@ public enum Scene {
         
         window.rootViewController = vc
         window.makeKeyAndVisible()
+        
+        /*
```

Comment:
The comment might be too wide which may introduce a horizontal scrolling. Maybe we could improve formatting in this case instead of adding a long comment?

Reply:
Done.